dca/tl_nc_notification.php

```diff
@@ -112,7 +112,7 @@
     'palettes' => array
     (
         '__selector__'                => array('type'),
-        'default'                     => '{title_legend},title,type',
+        'default'                     => '{title_legend},title,type;',
```

Comment:
It is not common in Contao to add a `;` at the end of a palette. If anyone would append to existing palettes, it should always be done with `;{…legend}…`